serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -6,11 +6,47 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-Knative Eventing uses the `default` broker unless otherwise specified.
+Knative provides a multi-tenant (MT) broker implementation that uses channels for event routing.
+An MT broker instance can be created by both users and cluster administrators.
 
-If you have cluster administrator permissions, you can create the `default` broker automatically using namespace annotation.
+.Prerequisites
+* Before you can create an MT broker implementation, you must create a channel. For more information, see xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using channels].
```

Comment:
`...you must create a channel.` is this the case? I think this is just defining the backing channel of the broker

Reply:
Will a broker be able to work / do anything with no channels though?
Trying to define the user story here, i.e. what is everything that's needed to actually use a broker?